tests/robot-tests/.env.example

```diff
@@ -8,11 +8,12 @@ ADMIN_EMAIL=ADMIN_EMAIL  # BAU1 user email
 ADMIN_PASSWORD=ADMIN_PASSWORD  # BAU1 user password
 ANALYST_EMAIL=ANALYST_EMAIL  # Analyst1 user email
 ANALYST_PASSWORD=ANALYST_PASSWORD  # Analyst1 user password
-RELEASE_COMPLETE_WAIT=120  # Time to wait for a release to publish
-WAIT_MEDIUM=120  # Variable used throughout tests
-WAIT_SMALL=45 # Variable used throughout tests
-WAIT_LONG=180  # Variable used throughout tests
-TIMEOUT=30 # Variable used throughout tests
-IMPLICIT_WAIT=15 # Variable used throughout tests
-FAIL_TEST_SUITES_FAST=1   # ("0" or "1") Fails entire test suite after any failure
+RELEASE_COMPLETE_WAIT=120  # Time to wait for a release to publish (in seconds)
+WAIT_MEMORY_CACHE_EXPIRY=5  # Time to wait for the in-memory cache to expire (in seconds)
```

Comment:
I don't mind it being at a value of some seconds, if only so you can manually test the memory cache locally with some browser refreshing.  However, I have lowered it to 2 seconds manually by default.

Reply:
I'm a bit confused because now in the local environment there's going to be an override for all entities of 10 seconds. Doesn't this need to be greater than 10 seconds? Just wondering if a change has not been pushed that I'm missing?

Reply:
Sorry, you got it exactly right and I hadn't committed that change to .env.example.